### ISSUE TITLE
Fix error with artificial node in the `Richmond_skeleton` network

### ISSUE THREAD

**Issue by tasseff** [NREL-SIIP/WaterModelsAnalytics.jl#26]
When simulating the `Richmond_skeleton` network, then using `get_node_dataframe` on all WaterModels node IDs, the following error is thrown for one of the IDs (whichever corresponds to `an55`, presumably):

```bash
ERROR: KeyError: key "an55" not found
Stacktrace:
[1] __getproperty at /home/btasseff/Projects/water-operation/.julia/packages/PyCall/BcTLp/src/PyCall.jl:307 [inlined]
[2] getproperty(::PyCall.PyObject, ::String) at /home/btasseff/Projects/water-operation/.julia/packages/PyCall/BcTLp/src/PyCall.jl:311
[3] _get_wntr_node_attribute(::PyCall.PyObject, ::String, ::String) at /home/btasseff/Projects/water-operation/WaterModelsAnalytics.jl/src/analysis/validation.jl:10
[4] get_node_dataframe(::Dict{String,Any}, ::Dict{String,Any}, ::PyCall.PyObject, ::PyCall.PyObject, ::String) at /home/btasseff/Projects/water-operation/WaterModelsAnalytics.jl/src/analysis/validation.jl:4
```

**Comment by jjstickel** [comment]
Can you share a feasible Richmond_skeleton solution with us so that we can test? Unless @enzoliuyang94 is able to get a solution with his Gurobi student-license.

**Comment by enzoliuyang94** [comment]
@jjstickel  I do have Gurobi installed and working. Let me try to run it and replicate the error. If @tasseff can directly send me the solution file, that would be nice too.

**Comment by tasseff** [comment]
I'm trying to produce one right now, but it's taking much longer than the Van Zyl network. I will send one as soon as I can.

**Comment by enzoliuyang94** [comment]
@tasseff For this error, you were using the hotfix-18 branch, not the master or yang branch, right?

**Comment by tasseff** [comment]
Correct!

**Comment by enzoliuyang94** [comment]
@jjstickel @tasseff The problem is identified, I can fix it, but I am not sure if this fix should be made to WaterModelAnalytics or WaterModels. Basically, for tank E, two artificial nodes were added, I believe the two artificial nodes are associated with the two pipes connected to tank E. However, no matter how many links are connected to a tank, only one artificial node is necessary and having just one artificial node and shutoff valve makes more physical sense to me. In my 'simulation.jl', I am assuming each tank to have exactly one artificial node. If we decide that each tank can have multiple artificial nodes, I can accomodate that as well.

**Comment by jjstickel** [comment]
Fixed with PR #25
